generator/generator_tool/generator_tool.cpp

```diff
@@ -245,6 +249,12 @@ int main(int argc, char ** argv)
   if (FLAGS_dump_feature_names != "")
     feature::DumpFeatureNames(datFile, FLAGS_dump_feature_names);
 
+  if (!FLAGS_srtm_path.empty())
+  {
+    string const mwmPath = my::JoinFoldersToPath(path, FLAGS_output + DATA_FILE_EXTENSION);
```

Comment:
Yes. Fixed. The code for calculation datFile is changed as well. It was:

string const datFile = path + FLAGS_output + DATA_FILE_EXTENSION;

and now it is

string const datFile = my::JoinFoldersToPath(path, FLAGS_output + DATA_FILE_EXTENSION);